bin/minify.js

```diff
@@ -89,13 +91,52 @@ function logAll(array) {
 
 function help() {
     const bin = require('../help');
-    const usage = 'Usage: minify [options]';
     
-    console.log(usage);
+    console.log('Usage:');
+    console.log('  minify [options]');
+    console.log('    This will read from stdin and output to stdout.');
+    console.log('  minify filePath [options]');
+    console.log('    This will read from a file and output to stdout.');
+    console.log('');
+    console.log('Examples:');
+    console.log('  minify -h');
+    console.log('    Print this message.');
+    console.log('');
+    console.log('  cat hello.js > minify --js > hello.min.js');
+    console.log('    Minify js as part of a pipeline of stream processors.');
+    console.log('');
+    console.log('  minify hello.html --htmlOptions="{\\"removeOptionalTags\\": false}" > hello.min.html');
+    console.log('    Minify an html file, writing it to a new file, without removing optional tags.');
+    console.log('');
     console.log('Options:');
     
     for (const name of Object.keys(bin)) {
         console.log('  %s %s', name, bin[name]);
     }
 }
 
+function processOptions(cliOptions) {
+    const options = {};
+    const files = [];
+
+    cliOptions.forEach((option) => {
+        const [key, val] = option.split('=');
+        switch (key) {
+            case '--htmlOptions':
+                options['html'] = JSON.parse(val);
+                break;
+            case '--imgOptions':
+                options['img'] = JSON.parse(val);
+                break;
+            case '--jsOptions':
+                options['js'] = JSON.parse(val);
+                break;
+            case '--cssOptions':
+                options['css'] = JSON.parse(val);
+                break;
+            default:
+                files.push(option)
+        }
+    })
+    return [files, options];
+}
```

Comment:
Could you add tests fot this and use `kebab-case` for options

Reply:
in 5ea30b5 I have:
Added some tessts, this required me to move this function to its own file so it could be imported directly into the tests.
Change the cli options to be kebab-case in all places